src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/metadata/ProjectSampleMetadataAjaxController.java

```diff
@@ -101,7 +100,7 @@ public ProjectSampleMetadataAjaxController(UIMetadataImportService metadataImpor
 	 * @param projectId identifier for the {@link Project} currently uploaded metadata to.
 	 */
 	@GetMapping("/upload/clear")
-	public void clearProjectSampleMetadata(HttpSession session, @RequestParam long projectId) {
```

Comment:
This actually should be a `DeleteMapping`

Reply:
That was an oversight. Please see 7a6200970002b1b687b352cbdad63a48ef3d7e5c.